docs/installation/migrate.rst

```diff
@@ -33,60 +36,165 @@ service. In this case, you migrate your existing self-managed |sr| to
 
+------------------------
+Migrate Schema Registry
+------------------------
 
-    :: 
+To migrate |sr| to |ccloud|, follow these steps:
+
+#.  Start the origin cluster.
+
+    If you are running a local cluster (for example, from a :ref:`ce-quickstart` download),
+    start only |sr| for the purposes of this tutorial.
     
-        schema.topic=_schemas
-        schema.registry.url=http://somehost:8081
-        schema.registry.client.basic.auth.credentials.source=USER_INFO
-        schema.registry.client.basic.auth.user.info=<user>:<password>
+    .. code:: bash
+  
+        <path-to-confluent>/bin/confluent start schema-registry
+        
   
+    .. tip:: The examples here show how to use a |crep| worker in *standalone mode* for schema migration.
+             In this mode, you cannot run |kconnect-long| and |crep| at the same time, 
+             because |crep| also runs |kconnect|. If you run |crep| in *distributed mode*, 
+             the setup is different and you do not have this limitation (you can use ``./bin/confluent start``). 
+             For more about configuring and running |kconnect| workers (includig |crep|) in standalone and  
+             distributed modes, see :ref:`Running Workers` <connect_userguide_standalone_config>` in the Connect guide.
+             
+                                    
+#.  Verify that ``schema-registry``, ``kafka``, and ``zookeeper`` are running.
+    
+    For example, run ``<path-to-confluent>/bin/confluent status``:
+    
+    ::
+    
+      schema-registry is [UP]
+      kafka is [UP]
+      zookeeper is [UP]
     
-#.  Start Replicator so that it can perform the one-time schema migration. 
+#.  Verify that no subjects exist on the destination |sr| in |ccloud|.
 
     .. code:: bash
+      
+        curl -u <schema-registry-api-key>:<schema-registry-api-secret> <schema-registry-url>/subjects
+        
+    If no subjects exist, your output will be empty (``[]``), which is what you want.
     
-        TBD example "start replicator" command
+    If subjects exist, delete them (TBD see Cloud Quick Start or https://docs.confluent.io/current/schema-registry/docs/develop/api.html to show them how to GET subjects and DELETE them.)
 
-#.  Stop all producers.
+#.  Set the destination |sr| to IMPORT mode.  For example: 
 
     .. code:: bash
+    
+        curl -u <schema-registry-api-key>:<schema-registry-api-secret> -X PUT -H "Content-Type: application/json" "https://<destination-schema-registry>:8081/mode" --data '{"mode": "IMPORT"}'
+        
+    .. tip:: If subjects exist on the destination |sr|, the import will fail with a message similar to this: ``{"error_code":42205,"message":"Cannot import since found existing subjects"}``
 
-        TBD example "stop producers" command
 
-#.  Wait until the replication lag is 0.
+#.  Configure a |crep| worker to specify the addresses of broker(s) in the destination cluster, as described in :ref:`config-and-run-replicator`.
```

Comment:
shouldn't use `(s)` to indicate plural ([per Style Guide](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation#StyleGuideforConfluentDocumentation-Terminology))